changelog.md

```diff
@@ -1,3 +1,50 @@
+# Version 1.1
+
+SafeLife v1.1 introduces many changes to make it easier to run SafeLife agents in a variety of settings. Some of these changes are backwards incompatible, so please read this carefully if you are upgrading from a previous version.
+
+## Changes to the core `safelife` package
+
+- Removed the `SafeLifeGame.performance_ratio()` method and replaced it with separate methods `points_earned()`, `available_points()`, and `required_points()`. Note that previously the points in `performance_ratio` were either zero or one for each cell, whereas now the full distribution of point types is used. This does not change the behavior in any benchmark levels since each benchmark level only used one type of point-awarding cell.
```

Comment:
"full distribution of point types" isn't clear at all...  suggest "proportion of the actual possible score, using a [full range of cell and goal types](https://github.com/PartnershipOnAI/safelife/blob/f86111950a6334aefb7369f700b2d76edcf72c9b/safelife/safelife_game.py#L572)"
(for copy & paste:
```proportion of the actual possible score, using a [full range of cell and goal types](https://github.com/PartnershipOnAI/safelife/blob/f86111950a6334aefb7369f700b2d76edcf72c9b/safelife/safelife_game.py#L572)
```